static/js/util/profile_edit.js

```diff
@@ -71,7 +73,68 @@ export function boundDateField(key, label, error) {
     <span className="validation-error-text">{error}</span>
   </div>;
 }
+export function boundEducationMonthField(keySet, label) {
+  const {updateProfile, profile} = this.props;
+  let onChange = month => {
+    let clone = Object.assign({}, profile);
 
+    let current_value = _.get(clone, keySet);
+        // format as ISO-8601
+    if(month === null){
+      _.set(clone, keySet, null);
+    }else if (current_value){
+      _.set(clone, keySet, moment(current_value).set('month', month.value).format('YYYY-MM-DD'));
```

Comment:
Can you create a constant for `YYYY-MM-DD` and replace the string in all places within the project where it is used?